app/src/lib/api.ts

```diff
@@ -374,6 +374,13 @@ export class API {
       return await parsedResponse<IAPIRepository>(response)
     } catch (e) {
       if (e instanceof APIError) {
+        if (org) {
+          throw new Error(
+            `Unable to create repository for organization '${
+              org.login
+            }' - Verify it exists and that you have permission to create a repository there.`
```

Comment:
:art: drop the `V` here from `Verify` or use a `.` instead of a `-` to indicate it's a new sentence.

Reply:
Just to clarify, do you mean to change the V to not be capitalized? Would this be preferable to the period I have added in the most recent commit?